content/blog/2022/q3-update/index.md

```diff
@@ -0,0 +1,95 @@
+- COESSING workshop: [infrastructure#1516](https://github.com/2i2c-org/infrastructure/issues/1516)
+- Eddy Symposium: [infrastructure#467](https://github.com/2i2c-org/team-compass/issues/467)
+- Allen Institute Summer Workshop on the Dynamic Brain [infrastructure#1621](https://github.com/2i2c-org/infrastructure/issues/1621)
+
+For a recap of one of these events, see our recent [blog post on the Jack Eddy symposium](https://2i2c.org/blog/2022/eddy-symposium-report).
+
+## Organization wide updates
+
+These are large-scale organizational and strategic efforts that impact all of 2i2c.
+
+**We applied for a CZI Grant**: In partership with The Carpentries, CSCCE, MetaDocencia, Invest in Open Infrastructure, and OpenLifeScience, we [applied for a CZI grant to provide cloud infrastructure services to global communities](../czi-global-communities-proposal/index.md).
+
+**We grew our team**: We've hired two new team members to lead new major efforts with 2i2c. [**James Munroe**](https://2i2c.org/author/james-munroe/) will lead efforts around _community guidance and product design_, and [**Jim Colliander**](https://2i2c.org/author/jim-colliander/) will lead efforts around _partnerships and sustainability_. We also [updated our Hiring and Candidate Search documentation](https://github.com/2i2c-org/team-compass/issues/436) in the process.
+
+**We're refining our strategy**: We've begun a process of revisiting and refining our strategy after a year of major operations, see [our strategic update blog post for more information](../strategic-update/index.md).
+
+**We completed the [CSCCE](https://cscce.org) community management training**. Two of our team members (James and Sarah) both completed a several-week community management course that was offered in partnership with [CZI](chanzuckerberg.org).
+
+**Our team member Sarah began a part-time role as the JupyterHub Community Strategic Lead**. Sarah will be leading community strategy efforts within JupyterHub for the next two years thanks to a grant to the JupyterHub team from CZI. [Check out this issue to follow our progress](https://github.com/jupyterhub/team-compass/issues/536).
+
+
+<!-- 
+TODO: Only post this if we get our engineering salary bands questions resolved before this post goes live.
+
+**We are hiring an engineer**: Do you know an open source cloud engineer with experience in Kubernetes and has desire make research and education more impactful, accessible, and delightful? Check out our job ad [at ].
+-->
+
+## Service improvements
+
+We made a number of improvements to our cloud infrastructure and the processes around our service.
+Here's a brief breakdown:
+
+**We expanded our shared clusters to new cloud providers and regions**. We now have a shared clusters already deployed on Google Cloud Platform on `us-central1-b` and `europe-west2` and we will soon deploy a shared cluster on Amazon Web Services on `us-west-2`.
```

Comment:
I would avoid reporting about things we have not deployed yet, ie. AWS shared cluster.